.github/workflows/zxf-dry-run-extended-test-suite.yaml

```
@@ -62,6 +67,38 @@ defaults:
     shell: bash
 
 jobs:
+  parameters:
+    name: Workflow Parameters
+    runs-on: hiero-citr-linux-medium
+    outputs:
+      solo-version: ${{ steps.set-outputs.outputs.solo-version }}
+      helm-release-name: ${{ steps.set-outputs.outputs.helm-release-name }}
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@ec9f2d5744a09debf3a187a3f4f675c53b671911 # v2.13.0
+        with:
+          egress-policy: audit
+
+      - name: Print Params
+        run: |
+          echo "Commit SHA: ${{ inputs.commit_sha }}" | tee -a "${GITHUB_STEP_SUMMARY}"
+          echo "Branch Name: ${{ inputs.branch_name }}" | tee -a "${GITHUB_STEP_SUMMARY}"
+          echo "Enable Extended Test Suite: ${{ inputs.enable-extended-test-suite }}" | tee -a "${GITHUB_STEP_SUMMARY}"
+          echo "Enable Abbreviated Panel: ${{ inputs.enable-abbreviated-panel }}" | tee -a "${GITHUB_STEP_SUMMARY}"
+          echo "Enable Hedera Node JRS Panel: ${{ inputs.enable-hedera-node-jrs-panel }}" | tee -a "${GITHUB_STEP_SUMMARY}"
+          echo "Enable SDK TCK Regression Panel: ${{ inputs.enable-sdk-tck-regression-panel }}" | tee -a "${GITHUB_STEP_SUMMARY}"
+          echo "Enable Mirror Node Regression Panel: ${{ inputs.enable-mirror-node-regression-panel }}" | tee -a "${GITHUB_STEP_SUMMARY}"
+          echo "Enable JSON-RPC Relay Regression Panel: ${{ inputs.enable-rpc-relay-regression-panel }}" | tee -a "${GITHUB_STEP_SUMMARY}"
+          echo "Enable Block Node Regression Panel: ${{ inputs.enable-block-node-regression-panel }}" | tee -a "${GITHUB_STEP_SUMMARY}"
+          echo "Additional Parameters: ${{ inputs.additional-parameters }}" | tee -a "${GITHUB_STEP_SUMMARY}"
+          echo "Solo Version: ${{ fromJson(inputs.additional-parameters).solo-version || '' }}" | tee -a "${GITHUB_STEP_SUMMARY}"
+          echo "Helm Release Name: ${{ fromJson(inputs.additional-parameters).helm-release-name || '' }}" | tee -a "${GITHUB_STEP_SUMMARY}"
+      - name: Set Outputs
```

nit: add a newline between the steps.

```suggestion

      - name: Set Outputs
```